src/module/wallet/component/display/AddWalletCard/AddWalletCard.style.ts

```diff
@@ -13,13 +12,16 @@ export const AddText = styled(Typography, { textTransform: "uppercase" })(({ the
     fontWeight: "bold",
 }));
 
-export const AddWalletCardRoot = styled(View)(({ theme }) => ({
-    flex: 1,
-    height: "100%",
+export const AddWalletCardRoot = styled(Col)(({ theme }) => ({
     width: "100%",
-    backgroundColor: theme.palette.gray[500],
+    height: "100%",
+    padding: 24,
+    overflow: "hidden",
+    borderRadius: 20,
+    backgroundColor: theme.palette.secondary,
 }));
 
-export const ContentRoot = styled(Row, { justifyContent: "center", alignItems: "center", gap: 20 })(() => ({
+export const ContentRoot = styled(Col, { justifyContent: "center", gap: 20 })(() => ({
```

Comment:
El resto de styled components los podemos eliminar si no se usan, a no ser que se tengan que poner luego en alguna parte 🤔

Reply:
you are right:)